### PR TITLE
Fix strict lint voilations

### PR DESCRIPTION
**Pull request by devahmedshendy**

### CODE REVIEW

**Comment by devahmedshendy** [comment]
@leogdion The fixes here are just the simple fixes, for more fixes like body length, missing docs, I am working on different branch so we don't mess with this one.

File: Sources/SyndiKit/Formats/Media/Podcast/PodcastPerson+Role.swift [hunk 1]
```diff
@@ -16,14 +16,29 @@ extension PodcastPerson {
 
     init?(role: Role) {
```
Comment (leogdion) [inline Sources/SyndiKit/Formats/Media/Podcast/PodcastPerson+Role.swift:17]: go ahead and add SwiftLint disable for cyclomatic complexity and body length

File: Sources/SyndiKit/Formats/Media/Podcast/PodcastPerson+Role.swift [hunk 2]
```diff
@@ -65,13 +80,26 @@ extension PodcastPerson {
 
     private init(knownRole: KnownRole) {
```
Comment (leogdion) [inline Sources/SyndiKit/Formats/Media/Podcast/PodcastPerson+Role.swift:81]: go ahead and add SwiftLint disable for cyclomatic complexity

**Review by leogdion — CHANGES_REQUESTED** [review]
This is great. Just a few changes and then let's chat about next steps.

**Pull request merged**